Main_Window:
  ros__parameters:
    local_position_topic: "current_pose"
    global_position_topic: "current_global_pose"
    global_origin_topic: "global_origin"
    marker_list_topic: "marker_list"
    state_topic: "state"
    planner_status_topic: "planner_status"
    select_planner_service: "select_planner"
    pid_planner_name: "pid_planner"
    dwa_planner_name: "dwa_planner"
    pid_planner_enabled_service: "pid_planner_enabled"
    dwa_planner_enabled_service: "dwa_planner_enabled"
    pid_planner_status_topic: "pid_planner_status"
    dwa_planner_status_topic: "dwa_planner_status"
    found_marker_list_topic: "found_marker_list"
    add_marker_service: "add_marker"
    clear_markers_service: "clear_markers"
    reorder_marker_service: "reorder_marker"
    reorder_markers_service: "reorder_markers"
    edit_marker_service: "edit_marker"
    remove_marker_service: "remove_marker"
    insert_marker_service: "insert_marker"
    clear_found_markers_service: "clear_found_markers"
    drive_forward_service: "drive_forward"
    car_style_turning_service: "car_style_turning"